### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by sturkmen72**
Update documentation

### CODE REVIEW

File: doc/tutorials/core/adding_images/adding_images.markdown [hunk 1]
```diff
@@ -25,51 +25,14 @@ By varying \f$\alpha\f$ from \f$0 \rightarrow 1\f$ this operator can be used to
...
-
- /// Create Windows
- namedWindow("Linear Blend", 1);
+Source Code
+-----------
 
- beta = ( 1.0 - alpha );
- addWeighted( src1, alpha, src2, beta, 0.0, dst);
+Download the source code from
+[here](https://github.com/Itseez/opencv/tree/master/samples/cpp/tutorial_code/ImgProc/AddingImages.cpp).
```
Comment (sturkmen72) [inline doc/tutorials/core/adding_images/adding_images.markdown]: https://github.com/Itseez/opencv/tree/master/samples/cpp/tutorial_code/ImgProc/AddingImages.cpp
should move to https://github.com/Itseez/opencv/tree/master/samples/cpp/tutorial_code/core/AddingImages/AddingImages.cpp

**Comment by mshabunin** [comment]
@sturkmen72 , please merge or rebase your changes to the latest version.

**Comment by alalek** [comment]
Please use `git pull --rebase` (or something like `git rebase -i upstream/master`) to eliminate unnecessary changes which are already applied.
Also please replace "https://github.com/Itseez/opencv" link to "https://github.com/opencv/opencv".

**Comment by sturkmen72** [comment]
i am still working on this PR. i will solve the conflicts soon

**Pull request closed without merging**